service/routes/api.js

```diff
@@ -72,12 +73,20 @@ router.get(/^\/v2\/polyfill(\.\w+)(\.\w+)?/, (req, res) => {
 	res.set('Access-Control-Allow-Origin', '*');
 
 	const outputStream = polyfillio.getPolyfillString(params);
-	outputStream.pipe(res, {end: false});
-	outputStream.on('end', () => {
-		if (req.query.callback && req.query.callback.match(/^[\w\.]+$/)) {
-			res.write("\ntypeof "+req.query.callback+"==='function' && "+req.query.callback+"();");
+	const callbackWrapper = miss.through(function () {},
+		function (cb) {
+			if (req.query.callback && req.query.callback.match(/^[\w\.]+$/)) {
+				cb(null, "\ntypeof "+req.query.callback+"==='function' && "+req.query.callback+"();");
+			} else {
+				cb();
+			}
+		}
+	);
```

Comment:
Do we have to use transform streams in a pipe like this?  It seems to be complicating the logic to push the whole contents of the output through the jsonP-callback-adding stream, then pass the whole lot on again to the response stream.  Previously we were listening for the end of stream event and then writing to the output stream, without seeing any of the stream content, which strikes me as a slightly better pattern, but I don't know a lot about streams.